projectiles/double_laser.gd

```diff
@@ -0,0 +1,33 @@
+class_name EffectDoubleLaser
+extends ProjectileEffect
+
+var Projectile : PackedScene = preload("res://units/projectile.tscn")
+
+@export var laser_offset = Vector2(0, -50)
+const is_double_laser = true
+
+func check_not_double_laser(effect: PackedScene) -> bool:
+	# TODO:: Filter on type? This implementation is Bad
+	if "is_double_laser" in effect:
+		return not effect.is_double_laser
+
+	# TODO:: Hacky? but const on the node does not work for packed scenes
+	return effect.resource_path != "res://projectiles/double_laser.tscn"
```

Comment:
❓ Non-blocking: can we type check the effect? I guess we're loading the scene, but we could probably type check the parent node of the scene to be able to check against a custom `class_name`? 🤔